package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "configstore",
-  "version": "1.4.0",
+  "version": "2.0.0",
```

Comment:
Yes, but I have a script for that. And I might want to do other changes before doing a release.